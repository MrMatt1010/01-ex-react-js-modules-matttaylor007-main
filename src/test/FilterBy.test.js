import { render, cleanup } from "@testing-library/react";
import FilterBy from "../components/FilterBy";

afterEach(cleanup);

test("renders the FilterBy component as expected", () => {
  const { asFragment } = render(<FilterBy />);
  expect(asFragment()).toMatchSnapshot();
});
