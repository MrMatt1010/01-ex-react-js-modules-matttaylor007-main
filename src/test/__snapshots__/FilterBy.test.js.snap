// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the FilterBy component as expected 1`] = `
<DocumentFragment>
  <div
    class="filterBy"
  >
    <label
      for="category"
    >
      Filter by
    </label>
    <select
      id="category"
    >
      <option
        selected=""
        value="all"
      >
        All
      </option>
      <option
        value="accessories"
      >
        Accessories
      </option>
      <option
        value="sunglasses"
      >
        Sunglasses
      </option>
    </select>
  </div>
</DocumentFragment>
`;
